docs/LTS-Strategy.md

```diff
@@ -0,0 +1,75 @@
+# Summary
+[summary]: #summary
+
+This document defines the Long-term support (LTS) release strategy for Hyperledger Aries CloudAgent Python (Aca-Py). This document is inspired from the [Hyperledger Fabric Release Strategy](https://github.com/hyperledger/fabric-rfcs/blob/main/text/0005-lts-release-strategy.md). 
+
+Long-term support definition from wikipedia.org:
+
+> **Long-term support (LTS)** is a product lifecycle management policy in which a stable release of computer software is maintained for a longer period of time than the standard edition.
+
+> **LTS** applies the tenets of reliability engineering to the software development process and software release life cycle. Long-term support extends the period of software maintenance; it also alters the type and frequency of software updates (patches) to reduce the risk, expense, and disruption of software deployment, while promoting the dependability of the software.
+
+# Motivation
+[motivation]: #motivation
+
+Most of the Aca-Py users rely upon using the [Docker images](https://github.com/hyperledger/aries-cloudagent-python/pkgs/container/aries-cloudagent-python)  which are published nightly and along with the [releases](https://github.com/hyperledger/aries-cloudagent-python/releases). These images contain the project dependencies/libraries which needs constant security vulnerability monitoring and patches based on the updates available from opensource.
+
+This is one of the factors which motivated setting up the LTS releases which requires the docker images to be scanned regularly and patching them for vulnerabilies. 
+
+In addition to this, administrators can expect the following of a LTS release:
+
+- Stable and well-tested code
+- A list of supported RFCs and features for each LTS version from this [document](https://github.com/hyperledger/aries-cloudagent-python/blob/main/docs/features/SupportedRFCs.md).
+- Minimal set of feature additions and other changes that can easily be applied, reducing the risk of functional regressions and bugs
+
+Similarly, there are benefits to Aca-Py maintainers, code contributors, and the wider community:
+
+- New features and other changes can quickly be applied to the main branch, and distributed to the user community for trial, without impacting production deployments.
+- Community feedback on new features can be solicited and acted upon.
+- Bug fixes only need to be backported to a small number of designated LTS releases.
+- Extra tests (e.g. upgrade tests for non-subsequent versions) only need to be executed against a small number of designated LTS releases.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## Release cadence
+
+Aca-Py will use the pattern of major, minor and patch releases `major.minor.patch` e.g. 0.10.5, 0.11.1, 0.12.0, 0.12.1, 1.0.0, 1.0.1 etc.
```

Comment:
“uses”, not “will use”. You might want to add a “semver” reference here — https://semver.org/